internal/convert.go

```diff
@@ -311,57 +342,72 @@ func (conv *Conv) unexpected(u string) {
 	}
 }
 
-// statsAddRow increments the count of rows for 'srcTable' if b is
+// StatsAddRow increments the count of rows for 'srcTable' if b is
 // true.  The boolean arg 'b' is used to avoid double counting of
 // stats. Specifically, some code paths that report row stats run in
 // both schema-mode and data-mode e.g. statement.go.  To avoid double
 // counting, we explicitly choose a mode-for-stats-collection for each
 // place where row stats are collected. When specifying this mode take
 // care to ensure that the code actually runs in the mode you specify,
 // otherwise stats will be dropped.
-func (conv *Conv) statsAddRow(srcTable string, b bool) {
+func (conv *Conv) StatsAddRow(srcTable string, b bool) {
 	if b {
 		conv.stats.rows[srcTable]++
 	}
 }
 
-func (conv *Conv) statsAddRows(srcTable string, count int64) {
+// StatsAddRows increments the count of rows for 'srcTable' by the count specified
+func (conv *Conv) StatsAddRows(srcTable string, count int64) {
 	conv.stats.rows[srcTable] += count
 }
```

Comment:
Suggest dropping this function and just inlining the access to conv.stats.rows (assuming we make stats public) where the function is used.

Reply:
Agreed. Now that we are making Stats as public, functions except the one's using bool parameter are not required. I have made changes accordingly.